include/nn/ccr/sys.h

```diff
@@ -0,0 +1,112 @@
+#pragma once
+#include <wut.h>
+
+/**
+ * \defgroup nn_ccr_sys
+ * \ingroup nn_ccr
+ * (see nn::ccr)
+ * @{
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum CCRSysPairingState
+{
+    //! Pairing is complete / hasn't started yet
+    CCR_SYS_PAIRING_FINISHED        = 0,
+    //! Pairing is in progress
+    CCR_SYS_PAIRING_IN_PROGRESS     = 1,
+    //! Pairing timed out
+    CCR_SYS_PAIRING_TIMED_OUT       = 2,
+} CCRSysPairingState;
+
+void
+CCRSysInit(void);
+
+void
+CCRSysExit(void);
+
+/**
+ * Shutdown the DRC in slot 0.
+ * 
+ * \return
+ * 0 on success.
+ */
+int32_t
+CCRSysDRCShutdown(void);
+
+/**
+ * Invalidate the DRC pairing in slot 0.
+ * 
+ * \return
+ * 0 on success.
+ */
+int32_t
+CCRSysInvalidatePairing(void);
+
+/**
+ * Check if the DRC in the specified slot is attached.
+ * 
+ * \param drcSlot
+ * Slot from 0 to 1.
+ * 
+ * \return
+ * \c TRUE if attached.
+ */
+BOOL
+__CCRSysDRCIsAttached(uint32_t drcSlot);
```

Comment:
Maybe use a enum for the slot? (Something similar to `OSScreenID`?)

Reply:
Yeah I was thinking about that, but this slot 0/1 logic is used in several files throughout #265 and didn't want to create an enum for each of them.